pkg/activator/handler/handler_test.go

```diff
@@ -567,62 +614,10 @@ func sendRequests(count int, namespace, revName string, respCh chan *httptest.Re
 	}
 }
 
-// getRT returns a roundTripper. probeErr if set is an error response returned to probe requests. probeCode is the
-// response code of probe requests. probeResp if set is the body to respond with for probe requests.
-// err if set is an error response returned for forwarded requests. body is the response body of forwarded requests.
-// wantHost if set is the host header required in forwarded requests.
-// lockerCh controls when responses are sent
-func getRT(t *testing.T, probeErr error, probeCode int, probeResps []string, err error, body string, wantHost string, lockerCh chan struct{}) network.RoundTripperFunc {
-	return network.RoundTripperFunc(func(r *http.Request) (*http.Response, error) {
-		if r.Header.Get(network.ProbeHeaderName) != "" {
-			if probeErr != nil {
-				return nil, probeErr
-			}
-			fake := httptest.NewRecorder()
-			// Make sure the probe is attributed with correct header.
-			if r.Header.Get(network.ProbeHeaderName) != queue.Name {
-				fake.WriteHeader(http.StatusBadRequest)
-				fake.WriteString("probe sent to a wrong system")
-				return fake.Result(), nil
-			}
-			fake.WriteHeader(probeCode)
-			resp := queue.Name
-			if len(probeResps) > 0 {
-				resp = probeResps[0]
-				probeResps = probeResps[1:]
-			}
-			fake.WriteString(resp)
-			return fake.Result(), nil
-		}
-		// Actual request test handler.
-		if err != nil {
-			return nil, err
-		}
-
-		// Now verify that the request has the required rewritten host header.
-		if got, want := r.Host, ""; got != want {
-			t.Errorf("The r.Host has not been cleared out, was: %q", got)
-		}
-		if got, want := r.Header.Get("Host"), ""; got != want {
-			t.Errorf("The Host header has not been cleared out, was: %q", got)
-		}
-
-		if wantHost != "" {
-			if got, want := r.Header.Get(network.OriginalHostHeader), wantHost; got != want {
-				t.Errorf("The %s header = %q, want: %q", network.OriginalHostHeader, got, want)
-			}
-		}
-
-		if lockerCh != nil {
-			lockerCh <- struct{}{}
-		}
-
-		fake := httptest.NewRecorder()
-
-		fake.WriteHeader(http.StatusOK)
-		fake.WriteString(body)
-		return fake.Result(), nil
-	})
+// getHandler returns an already setup activationHandler. The roundtripper is controlled
```

Comment:
it feels that the comment does not reflect reality?
Nor are parameters even used.

Reply:
Yep, I should delete this altogether